codelists/+partyRole.csv

```diff
@@ -5,3 +5,11 @@ processContactPoint,Process contact point,A contact point dedicated to this cont
 reviewContactPoint,Review contact point,The service from which information about the review procedure can be obtained.
 selectedParticipant,Selected participant,A party that has already been selected to participate in the design contest.
 informationService,Information service,An organization that can provide information regarding specific aspects of the contracting process.
+procurementServiceProvider,Procurement service provider,An organization that provides procurement services to contracting authorities.
+eSender,TED eSender,An organization that provides data transfer services to contracting authorities.
```

Comment:
```suggestion
procurementServiceProvider,Procurement service provider,An organization that provides procurement services to buyers.
eSender,TED eSender,An organization that provides data transfer services to buyers.
```

Reply:
Also just realised there's missing codes from this that should have been added:
* leadBuyer (OPP-050)
* awardingCentralPurchasingBody (OPP-051)
* acquiringCentralPurchasingBody (OPP-052)
* leadTenderer (OPT0170)
* evaluationBody (OPT-301)
* submissionReceiptBody (OPT-301)

Although I think this is related to the conversation in https://github.com/open-contracting/standard/issues/1180#issuecomment-1579627232 about whether these codes should go into this extension or 1.2

Reply:
I see all those codes in the diff: https://github.com/open-contracting-extensions/ocds_eu_extension/pull/12/files#diff-29c1a1808f67ab087059d6987ffa9780bd9f56dac6da2cffa39269ffb3dc0a69

Reply:
I see them now too, I must have not pulled the latest update...